README.md

```diff
@@ -39,7 +39,7 @@ If you want to test a whole Drupal site, and not an individual module, see
 
 1. `cd` to the directory with your Drupal module. Make sure it's a git
    repository first!
-1. `curl -L https://github.com/deviantintegral/drupal_tests/raw/master/setup.sh | bash`
+1. `bash -c "$(curl -fsSL https://github.com/deviantintegral/drupal_tests/raw/master/setup.sh)"`
```

Comment:
Fancy! Any advantages to this over piping?

Reply:
My understanding is that technically this avoids the need to wrap a function at all, because the curl command has to complete entirely before bash is invoked. However, wrapping in a function is still useful given our prior docs.

Reply:
Fancy! I learnt a few things about those curl options.